doh-client/client.go

```diff
@@ -43,6 +43,8 @@ import (
 	"github.com/miekg/dns"
 	"golang.org/x/net/http2"
 	"golang.org/x/net/idna"
+
+	"crypto/tls"
```

Comment:
Run `gofmt` please. It should sort the imports alphabetically, standard library first, third-party library last.

Reply:
done, rearranged also the whole list